docs/source/mcmc_samplers/slice_rank_shrinking_mcmc.rst

```diff
@@ -0,0 +1,7 @@
+**************************************************
+Slice Sampling - Rank Shrinking MCMC
+**************************************************
+
```

Comment:
Think for some reason the asterixes need to be the same length as the text. So should stop on the "C" of MCMC.